pkg/security/ebpf/c/dentry.h

```diff
@@ -36,7 +36,7 @@ struct bpf_map_def SEC("maps/pathnames") pathnames = {
     .type = BPF_MAP_TYPE_LRU_HASH,
     .key_size = sizeof(struct path_key_t),
     .value_size = sizeof(struct path_leaf_t),
-    .max_entries = 32000,
+    .max_entries = 64000,
```

Comment:
Should this be a const or configurable?

Reply:
It's pretty hard to make it configurable as it's used at compile time. That could be a constant using a #define but it's pretty much equivalent